eth/eventhandler/handlers.go

```diff
@@ -422,6 +423,13 @@ func (eh *EventHandler) handleClusterReactivated(txn basedb.Txn, event *contract
 		return nil, fmt.Errorf("could not process cluster event: %w", err)
 	}
 
+	// bump slashing protection for operator reactivated validators
+	for _, share := range toReactivate {
+		if err := eh.keyManager.(ekm.StorageProvider).BumpSlashingProtection(share.SharePubKey); err != nil {
```

Comment:
I don't really like this cast but I understand otherwise it will require spec change. I thought maybe to use add share and add the functionality in add share to bump if the share exist, it will not return error if it exists on validatorAdded though which might be an issue. wdyt? @moshe-blox @olegshmuelov

Reply:
the problem is that addshare currently does not bump the SP data if the account already exists (as it always did)